lua/lsp/utils.lua

```diff
@@ -10,19 +10,51 @@ function M.is_client_active(name)
   return false
 end
 
--- FIXME: this should return a list instead
-function M.get_active_client_by_ft(filetype)
-  if not lvim.lang[filetype] or not lvim.lang[filetype].lsp then
-    return nil
-  end
+function M.disable_formatting_capability(client)
+  -- FIXME: figure out a reasonable way to do this
+  client.resolved_capabilities.document_formatting = false
+  require("core.log"):debug(string.format("Turning off formatting capability for language server [%s] ", client.name))
+end
 
+function M.get_active_client_by_ft(filetype)
```

Comment:
It's because you can "filter" the list by other attributes, so this sort of a core neovim idiom, hence `by_ft`
I wanted to actually remove this function in favor of the one I added in https://github.com/neovim/nvim-lspconfig/blob/9ca394ec3a9305a6c30018ee6f731c9b23a4b25e/lua/lspconfig/util.lua#L345, but I need your help cleaning up the null-ls module 😄

Reply:
`list_acive_clients_by_ft` or `filter_active_clients_by_ft`